Added: example to exclude IP addresses from the campaign.

Abhinickz opened the pull request:
Fix: https://github.com/googleads/google-ads-perl/issues/97 Added: example to exclude IP addresses from the campaign by updating `campaign_criterion`.

laurachevalier4 commented on examples/campaign_management/update_campaign_criterion_ip_block.pl:
```diff
@@ -0,0 +1,149 @@
+      ->new({
+        create => $campaign_criterion,
+
+        # for removing the IP from the campaign criterion
```
Can you update this comment to say:
```
# To remove the IP block campaign criterion, use:
# remove => <campaign_criterion_resource_name>
```

laurachevalier4 commented on examples/campaign_management/update_campaign_criterion_ip_block.pl:
```diff
@@ -0,0 +1,149 @@
+    push @{$operations}, $campaign_criterion_operation;
+  }
+
+  # Issue a mutate request to update the excluded IPs of campaign criterion.
```
Can you update this to:
```
# Issue a mutate request to create the campaign criteria for the IP addresses to exclude.
```

laurachevalier4 reviewed (changes requested):
Thanks for this PR! Just a few changes before it can be merged.

Abhinickz commented on examples/campaign_management/update_campaign_criterion_ip_block.pl:
```diff
@@ -0,0 +1,149 @@
+use Google::Ads::GoogleAds::Client;
+use Google::Ads::GoogleAds::Utils::GoogleAdsHelper;
+use Google::Ads::GoogleAds::Utils::FieldMasks;
+use Google::Ads::GoogleAds::V12::Resources::CampaignCriterion;
```
Updated `V12` to `V13` in this example file.

Abhinickz commented on examples/campaign_management/update_campaign_criterion_ip_block.pl:
```diff
@@ -0,0 +1,149 @@
+# Running the example with -h will print the command line usage.
+my $customer_id  = "INSERT_CUSTOMER_ID_HERE";
+my $campaign_id  = "INSERT_CAMPAIGN_ID_HERE";
+my $CRITERION_ID = "27";                        # ip_block
```
ran `perltidy -pro=.perltidyrc examples/campaign_management/update_campaign_criterion_ip_block.pl`

Abhinickz commented:
Thanks [laurachevalier4](https://github.com/laurachevalier4) for the PR review.

I have updated the PR with the requested changes.
Please have a look and let me know if anything else needs to be updated.

laurachevalier4 commented on examples/campaign_management/update_campaign_criterion_ip_block.pl:
```diff
@@ -0,0 +1,150 @@
+my $customer_id  = "INSERT_CUSTOMER_ID_HERE";
+my $campaign_id  = "INSERT_CAMPAIGN_ID_HERE";
+
+# ip_block_criterion_id
```
To avoid confusion that there's some field called `ip_block_criterion_id`, can you update this to `ip_block criterion ID`?

laurachevalier4 commented on examples/campaign_management/update_campaign_criterion_ip_block.pl:
```diff
@@ -0,0 +1,150 @@
+        ipBlock      => {
+          ip_address => $ip,
+        },
+
```
For consistency with other examples, can you remove this blank line? (And on lines 76, 79, and 95)?

Abhinickz commented:
[laurachevalier4](https://github.com/laurachevalier4), Updated `ip_block_criterion_id` to `ip_block criterion ID` and removed blank lines for consistency with other examples.
Please review.
